src/components/fieldHelpers/label.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from "react"
+
+interface Props {
+  label?: string
+  required?: boolean
+  htmlFor?: string
```

Comment:
Instead of using `for` we can also wrap the input in the label tag:

```html
<label>
   I am label
   <input name="test" type="text" />
</label>
```
Maybe this approach would simplify the interface?

Reply:
I don't know because for me, we need to be able to choose to add a label or not, depending whether the component is used as a filter or not.